ios/RNNordicDfu.m

```diff
@@ -220,24 +220,29 @@ - (void)logWith:(enum LogLevel)level message:(NSString * _Nonnull)message
 
         NSURL * url = [NSURL URLWithString:filePath];
 
-        DFUFirmware * firmware = [[DFUFirmware alloc] initWithUrlToZipFile:url];
-
-        DFUServiceInitiator * initiator = [[[DFUServiceInitiator alloc]
-                                            initWithCentralManager:centralManager
-                                            target:peripheral]
-                                           withFirmware:firmware];
-
-        initiator.logger = self;
-        initiator.delegate = self;
-        initiator.progressDelegate = self;
-        initiator.alternativeAdvertisingNameEnabled = alternativeAdvertisingNameEnabled;
-
-        // Change for iOS 13
-        initiator.packetReceiptNotificationParameter = 1; //Rate limit the DFU using PRN.
-        [NSThread sleepForTimeInterval: 2]; //Work around for being stuck in iOS 13
-        // End change for iOS 13
-
-        DFUServiceController * controller = [initiator start];
+        @try {
+          DFUFirmware * firmware = [[DFUFirmware alloc] initWithUrlToZipFile:url];
+
+          DFUServiceInitiator * initiator = [[[DFUServiceInitiator alloc]
+                                              initWithCentralManager:centralManager
```

Comment:
This function is deprecated it should be updated [See iOSDFULibrary code](https://github.com/NordicSemiconductor/IOS-DFU-Library/blob/ec5364755f4fcdf68d62ff4cf796d22e7b935f40/iOSDFULibrary/Classes/Implementation/DFUServiceInitiator.swift#L335)